### PR TITLE
Feature/mys 183 os ver stats

### PR DESCRIPTION
**Pull request by shroomist**
in addition to platform, we're now sending osName and os.release, along with identity.

### CODE REVIEW

File: src/app/mysterion.js [hunk 1]
```diff
@@ -198,6 +196,7 @@ class Mysterion {
     this._messageBus = new IpcMessageBus(ipc)
     this._communication = new MainMessageBusCommunication(this._messageBus)
     this._communication.onCurrentIdentityChange((identityChange: CurrentIdentityChangeDTO) => {
+      this._startupEventTracker.sendEvent(identityChange.id)
```
Comment (donce) [inline src/app/mysterion.js]: !! This event will be triggered after each `VpnInitializer` cycle - this means that after sleep or random `mysterium_client` error, we'll receive another `startup` event - we should ensure this event is sent only once.

File: src/app/statistics/startup-event-tracker.js [hunk 2]
```diff
@@ -32,8 +33,13 @@ class StartupEventTracker {
...
   }
 
-  async sendEvent () {
-    const context = { platform: os.platform() }
+  async sendEvent (uid: string) {
+    const context = {
+      platform: os.platform(),
+      osName: osName(os.platform(), os.release()),
+      release: os.release(),
+      identity: uid
```
Comment (donce) [inline src/app/statistics/startup-event-tracker.js]: Why calling it `uid` and not `identity`, as it's called elsewhere?

File: src/app/statistics/startup-event-tracker.js [hunk 3]
```diff
@@ -32,8 +33,13 @@ class StartupEventTracker {
     this._eventSender = eventSender
   }
 
-  async sendEvent () {
-    const context = { platform: os.platform() }
+  async sendEvent (uid: string) {
+    const context = {
+      platform: os.platform(),
+      osName: osName(os.platform(), os.release()),
+      release: os.release(),
```
Comment (donce) [inline src/app/statistics/startup-event-tracker.js]: ?? Name seems confusing - it looks more like application release, not OS release. `osRelease` / `platformRelease`?

File: src/app/statistics/startup-event-tracker.js (hunk 2 above)
Comment (donce) [inline src/app/statistics/startup-event-tracker.js]: !! We should extend specs to test that new values are being sent.

File: test/unit/app/communication/main-message-bus-communication.spec.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  let mainCommunication
+  const callback = () => {}
+
+  beforeEach(() => {
+    messageBus = new SubscribableMessageBus()
+    mainCommunication = new MainMessageBusCommunication(messageBus)
+  })
+
+  describe('.onCurrentIdentityChangeOnce', () => {
+    it('fires once only', () => {
```
Comment (ignasbernotas) [inline test/unit/app/communication/main-message-bus-communication.spec.js]: ?? seems like you're not testing multiple event triggers, even though the test says so, or am I missing something?:)

File: src/app/mysterion.js [hunk 5]
```diff
@@ -197,6 +195,11 @@ class Mysterion {
     const ipc = new MainIpc(send, this._bugReporter.captureErrorException)
     this._messageBus = new IpcMessageBus(ipc)
     this._communication = new MainMessageBusCommunication(this._messageBus)
+
```
Comment (ignasbernotas) [inline src/app/mysterion.js]: Mysterion class is growing bigger and bigger, maybe it's time to think about moving out new parts out of it somehow? Just a thought:)

File: test/unit/app/communication/main-message-bus-communication.spec.js (hunk 4 above)
Comment (shroomist) [reply test/unit/app/communication/main-message-bus-communication.spec.js]: yes, this test makes sure that no callbacks are remaining once we fire the related event.

File: src/app/mysterion.js (hunk 5 above)
Comment (shroomist) [reply src/app/mysterion.js]: we've registered a tech debt tasks related to that.
we might start deconstructing as soon as we're done with the sprint tickets (pretty possible this week)

File: test/unit/app/communication/main-message-bus-communication.spec.js (hunk 4 above)
Comment (ignasbernotas) [reply test/unit/app/communication/main-message-bus-communication.spec.js]: Ah, got it, cheers

File: src/app/communication/main-message-bus-communication.js [hunk 6]
```diff
@@ -123,6 +123,14 @@ class MainMessageBusCommunication implements MainCommunication, MapSyncCommunica
     this._on(messages.CURRENT_IDENTITY_CHANGED, callback)
   }
 
+  onCurrentIdentityChangeOnce (callback: (CurrentIdentityChangeDTO) => void) {
+    let cb = (currentIdentityChange: CurrentIdentityChangeDTO) => {
```
Comment (donce) [inline src/app/communication/main-message-bus-communication.js]: `const`

**Pull request merged**